lib/vmdb/console_methods.rb

```diff
@@ -33,6 +33,7 @@ def simulate_queue_worker(break_on_complete: false, quiet_polling: true)
         end
         if q
           puts "\e[33;1m\n** Delivering #{MiqQueue.format_full_log_msg(q)}\n\e[0;m"
+          q.update!(:state => MiqQueue::STATE_DEQUEUE, :handler => MiqServer.my_server)
```

Comment:
Not sure where `#handler` is being used but I thought it best to stay consistent with https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_queue.rb#L283